builder/amazon/ebsvolume/builder.go

```diff
@@ -137,7 +137,9 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, []string, error) {
 	}
 
 	packer.LogSecretFilter.Set(b.config.AccessKey, b.config.SecretKey, b.config.Token)
-	return nil, warns, nil
+
+	generatedData := []string{"SourceAMIName"}
+	return generatedData, warns, nil
 }
```

Comment:
Do you want to add the generatedData to the artifact here, too?

Reply:
ops, forgot this one because uses its own artifact implementation...thx!!

Reply:
@sylviamoss, I think will we also need `generatedData` for the other builders, am I right ?

Reply:
Yes @azr, but I think it's much more code and it will be easier to review in a new PR.
The way that it is right now won't affect the current behavior of other builders but it won't include the feature as well.